Gemfile

```diff
@@ -20,6 +20,7 @@ gem 'business_time'
 gem 'holidays'
 gem 'sentry-raven'
 gem 'bootstrap-sass'
+gem 'jira-ruby'
```

Comment:
let's add require: false, and load it only where needed (ideally in service class)